File: Task/FizzBuzz/Java/fizzbuzz.java
class FizzBuzz {

    public static void main(String[] args) {

        for (int i = 1; i < 101; i++) {
            if ((i % 3 == 0) && (i % 5 == 0)) {
                System.out.print("'fizz buzz', ");
            } else if (i % 3 == 0) {
                System.out.print("'fizz', ");
            } else if (i % 5 == 0) {
                System.out.print("'buzz', ");
            } else {
                System.out.printf("%d, ", i);
            }
        }
    }
}

File: Task/FizzBuzz/Zig/fizzbuzz.zig
const print = @import("std").debug.print;
pub fn main() void {

    for(1..101) |i| {
        if (i % 3 == 0 and i % 5 == 0) {
            print("FizzBuzz\n", .{});
        } else if (i % 3 == 0) {
            print("Fizz\n", .{});
        } else if (i % 5 == 0) {
            print("Buzz\n", .{});
        } else {
            print("{}\n", .{i});
        }
    }
}
